browser/ephemeral_storage/ephemeral_storage_forget_by_default_browsertest.cc

```diff
@@ -570,4 +570,57 @@ IN_PROC_BROWSER_TEST_F(EphemeralStorageForgetByDefaultDisabledBrowserTest,
   EXPECT_EQ(1u, GetAllCookies().size());
 }
 
+class EphemeralStorageForgetByDefaultIncognitoBrowserTest
+    : public EphemeralStorageForgetByDefaultBrowserTest {
+ public:
+  EphemeralStorageForgetByDefaultIncognitoBrowserTest() {
+    scoped_feature_list_.InitAndEnableFeature(
+        net::features::kBraveForgetFirstPartyStorage);
+  }
+  ~EphemeralStorageForgetByDefaultIncognitoBrowserTest() override = default;
+
+  void SetUpCommandLine(base::CommandLine* command_line) override {
+    EphemeralStorageForgetByDefaultBrowserTest::SetUpCommandLine(command_line);
+    if (IsPreTest()) {
+      command_line->AppendSwitch(switches::kIncognito);
+    }
+  }
+
+  static bool IsPreTest() {
```

Comment:
nit: I think this should be called `IsPreTestToEnableIncognito`, because you have 2 `PRE_` tests, but you're actually interested in one of them.